patches/tools-grit-grit-gather-policy_json.py.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/tools/grit/grit/gather/policy_json.py b/tools/grit/grit/gather/policy_json.py
+index dc50b9f5e68e2616810e9f451c2dbdaf94a0fe8e..a3c4b800412f3e8229b72401f28128764beed87c 100644
+--- a/tools/grit/grit/gather/policy_json.py
++++ b/tools/grit/grit/gather/policy_json.py
+@@ -330,3 +330,5 @@ class PolicyJson(skeleton_gatherer.SkeletonGatherer):
+       }
+     else:
+       raise Exception('Unknown build')
++
```

Comment:
don't add empty lines pls.

Reply:
Resolved in 43bf0e0276e8c4291dc0b380643f1d4d090df11d.